api/Window.json

```diff
@@ -2789,6 +2789,42 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "relative-multi-screen": {
```

Comment:
I think we can drop this section, moveBy's parameters are relative offsets from the window's current position, not absolute coordinates in a single- or multi-screen space.

Reply:
OK, makes sense. Removed.